src/xo-app/host/tab-advanced.js

```diff
@@ -39,6 +39,14 @@ export default ({
           labelId='enableHostLabel'
         />
       }
+      <TabButton
+        btnStyle='danger'
+        handler={detachHost}
+        handlerParam={host}
+        icon='host-eject'
+        labelId='detachHost'
+        redirectOnSuccess='/'
```

Comment:
But it would have been better to handle this like we did for VM deletion: at the level page, redirect to the home screen when the object disappears.

@pdonias Could you do that?